assets/section-main-product.css

```diff
@@ -298,6 +306,15 @@ fieldset.product-form__input .form__label {
 
 /* Product info */
 
+.different-color-scheme {
+  padding: 2rem;
+  margin-top: 1.5rem;
+}
+
+.different-color-scheme.product__info-container > *:first-child {
+  margin-top: 0;
+}
+
```

Comment:
One other thing we will need to adjust is the `top` value for the `product__info-container--sticky` when the color scheme are different. 
Here is an [video explanation](https://screenshot.click/26-27-1tlnp-m6mb8.mp4)

Reply:
Thanks Ludo for mention it!